hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptor.java

```diff
@@ -275,11 +275,11 @@
 
   /**
    * Check if normalization enable flag of the table is true. If flag is false
-   * then no region normalizer won't attempt to normalize this table.
+   * then region normalizer won't attempt to normalize this table.
    *
-   * @return true if region normalization is enabled for this table
+   * @return value of NORMALIZATION_ENABLED key for this table if present else return defaultValue
    */
-  boolean isNormalizationEnabled();
+  boolean isNormalizationEnabled(boolean defaulValue);
```

Comment:
This is the only place we use this pattern to get a config in TableDescriptor? I uggest we align these methods with the same pattern.

Reply:
That make sense. 
I was trying to define a logical definition about what will override the other. So currently we have a default value for table descriptor. What i am trying to do is remove it from the default list and define default at hbase-site level. So going forward - either this property will be present or not. and if not present then hbase-site value will be used and if present then the value defined by table descriptor will be used.

Let me fix this pattern in code and make it clear.